configs/localization/ssn/README.md

```diff
@@ -37,12 +37,34 @@ You can use the following command to train a model.
 python tools/train.py ${CONFIG_FILE} [optional arguments]
 ```
 
-Example: train SSN model on thumos14 dataset.
+Example: train SSN on THUMOS14 dataset.
 
 ```shell
 python tools/train.py configs/localization/ssn/ssn_r50_450e_thumos14_rgb_train.py
 ```
 
+Example: train SSN on ActivityNet-1.3 dataset.
+
+1. Get the result of action proposals. (Take proposal generator BMN as an example.)
+
+```shell
+python tools/test.py configs/localization/bmn/bmn_400x100_2x8_9e_activitynet_feature.py checkpoints/SOME_CHECKPOINT.pth --eval AR@AN --out results.json
+```
+
+2. Convert the output proposal file of proposal generator (BSN, BMN) into the input proposal file of action classifier. (Currently supports SSN and P-GCN, not including TSN, I3D etc.)
```

Comment:
> ... of proposal generator (BSN, BMN)

> Currently supports SSN and P-GCN, not including TSN, I3D etc.

What's the relation between these two?

Reply:
SSN and P-GCN predict categories and refine boundaries of proposals, while TSN and I3D only do classification.

Reply:
Let's try to improve the writting of this sentence.

> Convert the output proposal file of proposal generator (BSN, BMN) into the input proposal file of action classifier.

It mentions proposal generator, and BSN and BMN.

> (Currently supports SSN and P-GCN, not including TSN, I3D etc.)

Then this note mentions something else, such as SSN, P-GCN, ISN, I3D. From the perspective of an innocent reader, how are they related?

Reply:
The pipeline of multi-stage Temporal Action Localization:
1. Extract features with action recognizers/action classifiers (TSN, I3D etc.).
2. Generate proposals with proposal generators (BSN, BMN etc.).
3. Predict categories of proposals with action classifiers (SSN, P-GCN).

Reply:
This is much clearer now